cmd/tether/attach_linux.go

```diff
@@ -33,6 +33,48 @@ import (
 
 var backchannelMode = os.ModePerm
 
+func setTerminalSpeed(fd uintptr) error {
+	var current struct {
+		termios syscall.Termios
+	}
+
+	// get the current state
+	if _, _, errno := syscall.Syscall6(syscall.SYS_IOCTL,
+		uintptr(fd),
+		syscall.TCGETS,
+		uintptr(unsafe.Pointer(&current.termios)),
+		0,
+		0,
+		0,
+	); errno != 0 {
+		return errno
+	}
+
+	// copy it as the future
+	future := current.termios
+
+	// unset 9600 bps
+	future.Cflag &^= syscall.B9600
+	// set them to 115200 bps
+	future.Cflag |= syscall.B115200
+	future.Ispeed = syscall.B115200
+	future.Ospeed = syscall.B115200
```

Comment:
This is just setting the correct value on the linux side, I don't think there is a speed limit on the ESXi side. By the way how did you test it?